docs/kpod-exec.1.md

```diff
@@ -0,0 +1,50 @@
+% kpod(1) kpod-exec - Execute a command in a running container
+% Brent Baude
+# kpod-exec "1" "December 2017" "kpod"
+
+## NAME
+kpod-exec - Execute a command in a running container
+
+## SYNOPSIS
+**kpod exec**
+**CONTAINER**
+[COMMAND] [ARG...]
+[**--help**|**-h**]
+
+## DESCRIPTION
+**kpod exec** executes a command in a running container.
+
+
+## OPTIONS
+**-d, --detach=true|false**
+Not supported
```

Comment:
"Not currently supported" perhaps?  If we're planning to never support them then I think we shouldn't list them.

Reply:
I agree remove this.

Reply:
Remove

Reply:
fixed